js/util.js

```diff
@@ -134,19 +134,22 @@
 
     // angle between two vectors in radians
     function angle(v1, v2){
-        var diff = v1.rad - v2.rad;
+        var diff = v1.radians() - v2.radians();
         return atan2(sin(diff), cos(diff));
     }
 
     // Create a vector from an angle in degrees and a magnitude (length)
     function Vector(x,y){
-        this.x = x;
-        this.y = y;
+        this.x = Number(x);
+        this.y = Number(y);
```

Comment:
Should the type conversions be handled here or elsewhere in Waterbear? I'm assuming this is to convert strings from the input boxes into numbers, so that the vector always Numbers only.